changelog.md

```diff
@@ -0,0 +1,103 @@
+## Differences
```

Comment:
How do you see this file evolving over time? Should the items annotated below belong in a section of the changelog that allows for future discovery and updating older models if required? Is it worth bounding changelog updates to an intended release of go-oscal?

Reply:
Ive got some stories I plan on creating today. I would like the change log to be by release, open to additional thoughts, it is something I have done on other projects. I typically put all changes in the change log until release, then add the release and start putting any new changes. I think its something I would like to create in the contributers to formalize the manual process for now, then look into changelog tools later similar to the new schema update. 

As for updating/changing older models. I am not so sure. When generate changes the schema at least pre v1 those changes should be heavily documented, potentially avoided, idk if there is a point where older schemas just get solidified as far as their struct names etc. Now that all models are completely linked and there aren't any unlinked/mislinked models, I could see a case for no longer updating without external feedback, but I could also see keeping up to date with an Generate logic that could come up to maintain consistency.